index.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
 /**
- * Adds commas to a number
- * @param {number} number
- * @param {string} locale
+ * Push and pop from a stack
```

Comment:
This comment is for every single function ?

Reply:
Should these be broken down by function ? and if yes, where should these be added in the code.

Reply:
Right above the function, a top level comment to just tell what the class/library does but individual function should be commented